includes/definitions.inc.php

```diff
@@ -1003,6 +1003,17 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_frequency';
 $config['os'][$os]['over'][2]['text']  = 'Frequencies';
 
+$os = 'eatonups';
+$config['os'][$os]['text']             = 'Eaton UPS';
+$config['os'][$os]['type']             = 'power';
+$config['os'][$os]['icon']             = 'eaton';
```

Comment:
the new icon eatonups inon and eatonups vs eaton OS options was from my initial local changes before I even thought about attempting to merge..

It was mostly so I could see what I was changing / breaking while trying to have minimal impact on anything else..